docs/src/main/asciidoc/_admin-guide/management-tasks/YAML_Configuration_file.adoc

```diff
@@ -107,3 +107,33 @@ wildfly-configuration:
                 index: 0
 ----
 
+You can't configure everything in YAML, it doesn't currently support:
+
+ - extension: to add extension to the server as this might require modules which can be missing.
+ - deployment: to add *managed* deployments to the server as this requires more than just some configuration.
+ - deployment-overlay: to add deployment-overlays to the server as this requires more than just some configuration.
+ - path: since those should already have been defined when the YAML files are parsed.
+
+While *managed* deployments are not supported you can still define *unmanaged* deployments that are not meant to be managed using the management API which is fine since you are in *read-only* mode with YAML and your changes aren't persisted.
```

Comment:
This isn't truly accurate. These resources can still be manipulated using the management API. It's true that changes are not persisted but that's not the same thing.

While *managed* deployments are not supported you can still define *unmanaged* deployments. An unmanaged deployment is one where the server does not make an internal copy of the deployment content for its use; instead it directly uses the configured content. See [Managed and Unmanaged Deployments] (https://docs.wildfly.org/32/Admin_Guide.html#managed-and-unmanaged-deployments) for details.

except a proper adoc ref link instead of my lazy github ui syntax ;)